src/main/kotlin/com/openlattice/organizations/DatasetController.kt

```diff
@@ -158,12 +186,12 @@ class DatasetController : DatasetApi, AuthorizingComponent {
             @PathVariable(TABLE_NAME) tableName: String,
             @PathVariable(COLUMN_NAME) columnName: String
     ) {
-        val tableId = getExternalDatabaseObjectId(organizationId, tableName)
-        val columnId = getExternalDatabaseObjectId(tableId, columnName)
-        val aclKey = AclKey(organizationId, tableId, columnId)
+        val tableFqnToId = getExternalDatabaseObjectFqnToIdPair(organizationId, tableName)
```

Comment:
I think you could use the same code for deleting one column as for deleting multiple ones, since the checks and the underlying code after is also the same (`deleteOrganizationExternalDatabaseColumn` just calls `deleteOrganizationExternalDatabaseColumns`)
`deleteExternalDatabaseColumns(organizationId, tableName, setOf(columnName))`

Reply:
This would also imply to delete `deleteOrganizationExternalDatabaseColumn` from `ExternalDatabaseManagementService`, since it's only used here

Reply:
good call

Reply:
Sorry, I probably wasn't clear enough on this. I meant to just replace the whole function body with calling the function in this class:
```
@Timed
    @DeleteMapping(path = [ID_PATH + TABLE_NAME_PATH + COLUMN_NAME_PATH + EXTERNAL_DATABASE_COLUMN])
    override fun deleteExternalDatabaseColumn(
            @PathVariable(ID) organizationId: UUID,
            @PathVariable(TABLE_NAME) tableName: String,
            @PathVariable(COLUMN_NAME) columnName: String
    ) {
    deleteExternalDatabaseColumns(organizationId, tableName, setOf(columnName))
}
```